pages/dkp/kommander/2.2/dkp-upgrade/custom-apps/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Custom Applications
+title: Upgrade Custom Applications
+menuWeight: 60
+excerpt: Verify the compatibility of Custom Applications with the current Kubernetes version
+beta: false
+enterprise: true
+---
+
+We recommend upgrading your Custom Applications to the latest compatible version as soon as possible. Since there is no expectation of support by D2iQ for Custom Applications, you must upgrade them manually.
+
+<p class="message--warning"><strong>WARNING: </strong>Ensure you validate any Custom Applications you run for compatibility issues against the <a href="https://docs.d2iq.com/dkp/kommander/2.2/release-notes/">current Kubernetes version</a>. If the Custom Application’s version is not compatible with the Kubernetes version, do not continue with the Konvoy upgrade. Otherwise, your cluster’s services will stop running.
```

Comment:
```suggestion
<p class="message--warning"><strong>WARNING: </strong>Ensure you validate any Custom Applications you run for compatibility issues against the <a href="https://docs.d2iq.com/dkp/kommander/2.2/release-notes/">current Kubernetes version</a>. If the Custom Application’s version is not compatible with the Kubernetes version, do not continue with the Konvoy upgrade. Otherwise, your custom Applications may stop running.
```